captum/optim/_param/image/images.py

```diff
@@ -1,8 +1,8 @@
-import imageio
 import matplotlib.pyplot as plt
 import numpy as np
 import torch
 import torch.nn as nn
+from PIL import Image
```

Comment:
In terms of PIL, I think that our standard install setup might not have PIL package support. We can have this import in try-expect and ask the users to install it. See the example: https://github.com/pytorch/captum/pull/468/files#diff-0399fdfb934ad0d1e07d13305904829c89866b3c4adce935da861aff725d03fcR46

Reply:
Alright, I've added a try-expect block for alerting users that Pillow/PIL is required if it's not already installed!

Reply:
Thank you!